lib/matplotlib/quiver.py

```diff
@@ -374,7 +353,9 @@ def contains(self, mouseevent):
             return True, {}
         return False, {}
 
-    quiverkey_doc = _quiverkey_doc
+    @cbook.deprecated("3.2")
+    def quiverkey_doc(self):
```

Comment:
Does this work? Have you forgotton `@property`?

Reply:
indeed